TrafficCapture/trafficReplayer/src/main/resources/log4j2.properties

```diff
@@ -53,6 +53,19 @@ appender.TRANSACTION_SUMMARIES_LOGFILE.policies.time.modulate = true
 appender.TRANSACTION_SUMMARIES_LOGFILE.strategy.type = DefaultRolloverStrategy
 appender.TRANSACTION_SUMMARIES_LOGFILE.strategy.max = 720
 
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.type = RollingFile
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.name = AllActiveWorkMonitorFile
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.fileName = logs/longRunningActivity.log
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.filePattern = logs/%d{yyyy-MM}{UTC}/longRunningActivity-%d{yyyy-MM-dd-HH-mm}{UTC}-%i.log
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.layout.type = PatternLayout
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.layout.pattern = %msg ([%-5level] %d{yyyy-MM-dd HH:mm:ss,SSS}{UTC})%n
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.policies.type = Policies
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+appender.ALL_ACTIVE_WORK_MONITOR_LOGFILE.policies.time.interval = 60
```

Comment:
am I remembering correctly that the unit for the interval value here is the smallest unit of the `filePattern`? So this means every 60 minutes? This could clearly vary enormously, but do you have a sense of how large this file might be within that hour?

Reply:
60 represents minutes.

Assuming that we are rendering 64 scopes for each one of our 67 classes and that each line is about 200 bytes (this will be dependent upon the number of attributes, but 200 bytes is where we're at today), at 120 observations per file or per hour, we'd be at ~100MB as the theoretical max.  Typically, this file should be much smaller, with a couple span types being open for longer (channel, trafficStream, maybe even some responses or requests), but that should be < 10% of the total.  We may need to eventually supply the time edges as a command line argument (especially if one would want to disable the feature entirely).

This was all put into bigger files to minimize the number of times that a user would need to rerun tail -f as the underlying file handle changes.